src/reducers/index.js

```diff
@@ -1,22 +1,21 @@
 import { combineReducers } from 'redux';
-import { reducer as formReducer } from 'redux-form';
+import { reducer as formReducer, blur, focus } from 'redux-form';
 import { routerReducer } from 'react-router-redux';
-import { blur } from 'redux-form';
-import { blurFieldEntry } from 'src/reducers/formHelpers';
+import { getIn, updateIn } from 'zaphod/compat';
+
+import { transformFieldEntry } from 'src/reducers/formHelpers';
 import selectionReducer from 'src/reducers/selectionReducer';
 
 const blurType = blur().type;
-const applyFormReducer = (state, action) => {
-  if(action.type === blurType) {
-    return {
-      ...state,
-      values: {
-        ...state.values,
-        [action.meta.field]: blurFieldEntry( action.meta.field, action.payload )
-      },
-    };
+const focusType = focus().type;
```

Comment:
No, the do not export them.  I've looked at both their documentation and their source code and was unable to find anywhere where that is exposed.  Unless I missed something, this is the best way forward.

They prefix their actions with `@@redux-form/` which is defined but, again, not exported.